Userland/Libraries/LibRegex/RegexParser.cpp

```diff
@@ -1877,6 +1877,28 @@ bool ECMA262Parser::parse_capture_group(ByteCode& stack, size_t& match_length_mi
 {
     consume(TokenType::LeftParen, Error::InvalidPattern);
 
+    auto enter_capture_group_scope = [&] {
+        m_capture_groups_in_scope.empend();
+    };
+    auto exit_capture_group_scope = [&] {
+        auto last = m_capture_groups_in_scope.take_last();
+        m_capture_groups_in_scope.last().extend(move(last));
+    };
+    auto register_capture_group_in_current_scope = [&](auto identifier) {
+        m_capture_groups_in_scope.last().empend(identifier);
```

Comment:
Missing `move()`? `auto` makes it kinda hard to tell what this is (and I don't have CLion handy right now)

Reply:
it's either a `String` or a `size_t`.
I'll `move()` it, why not :shrug: